test/test_core.py

```diff
@@ -5868,7 +5868,7 @@ def test_fs_nodefs_readdir(self):
     self.cflags += ['-lnodefs.js']
     suffix = ''
     if self.get_setting('WASMFS'):
-      suffix = '.wasmfs'
+      suffix = '.wasmfs_win' if WINDOWS else '.wasmfs'
```

Comment:
IIUC the issue here is the order in which directories are listed? 

I wonder if we can instead have either:

1. Have wasmfs list directories in a deterministic / alphabetical order
2. Update the test to avoid depending on specific ordering.

Adding another separate copy of the expectations seems less desirable.

Reply:
This is not an ordering problem, but a problem that Windows does not support FIFO objects on the filesystem, so the test is unable to add one. And hence fails to `readdir()` it in the filesystem on Windows.

Reply:
check